src/Bicep.LangServer/Handlers/BicepCompletionHandler.cs

```diff
@@ -48,7 +61,7 @@ public override Task<CompletionItem> Handle(CompletionItem request, Cancellation
             DocumentSelector = DocumentSelectorFactory.Create(),
             AllCommitCharacters = new Container<string>(),
             ResolveProvider = false,
-            TriggerCharacters = new Container<string>(":", " ", ".")
+            TriggerCharacters = new Container<string>(":", " ", ".", "/")
```

Comment:
Does using `/` as a trigger character have any impact on the completions for resource types?

Reply:
It is not critical to have this trigger, and if there's too many potential false positive fires I prefer to remove it. That being said, I believe this trigger actually complements resource type completions well

Reply:
Update: This has _no_ affect on resource type completions since the intellisense is active throughout the typing process.

Reply:
Do resource type completions still work ok with it if the user escapes out of the completion and then types `/`?

---
In reply to: [557843332](https://github.com/Azure/bicep/pull/1236#discussion_r557843332) [](ancestors = 557843332)

Reply:
Btw, didn't notice any issues with this.

---
In reply to: [564874826](https://github.com/Azure/bicep/pull/1236#discussion_r564874826) [](ancestors = 564874826,557843332)